src/transformState.js

```diff
@@ -5,7 +5,27 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (action.type) {
```

Comment:
You can use destructurization here for action 
```suggestion
  for (const action of actions) {
    const {} = action;
      switch (action.type) {
```